modules/ROOT/pages/apikit-4-set-header-task.adoc

```diff
@@ -3,39 +3,81 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you add header information to the response that complies with the WSDL. Header information that complies with the WSDL document is called APIUsageInformation.
+Using APIkit you can set headers to your application's response based on the WSDL declarations.
```

Comment:
I suggest this wording "Use APIkit to set headers to your ..."
If you want to keep what you have, add a comma after APIkit

What do you mean by setting headers to your application's response?